src/osal/lv_chibios.c

```diff
@@ -0,0 +1,340 @@
+/**
+ * @file lv_chibios.c
+ *
+ * Port of LVGL OS abstraction layer using ChibiOS.
+ *
+ */
+
+#if LV_USE_OS == LV_OS_CHIBIOS
```

Comment:
`LV_*` macros are undefined.

```suggestion
/*********************
 *      INCLUDES
 *********************/
 
 #include "lv_chibios.h"

#if LV_USE_OS == LV_OS_CHIBIOS
```